mantle/kola/tests/crio/crio.go

```diff
@@ -357,6 +358,12 @@ func crioNetwork(c cluster.TestCluster) {
 func crioNetworksReliably(c cluster.TestCluster) {
 	m := c.Machines()[0]
 
+	// Figure out the host IP address on the crio default bridge. This is
+	// required as the default subnet was changed in 1.18 to avoid a conflict
+	// with the default podman bridge.
+	subnet := c.MustSSH(m, fmt.Sprintf("jq --raw-output '.ipam.ranges[0][0].subnet' /usr/etc/cni/net.d/100-crio-bridge.conf"))
```

Comment:
This seems...fragile, and I am honestly somewhat skeptical that we should continue to carry this test versus investing in things like running crio's upstream test suite, and kind of more importantly crio is an implementation detail of Kubernetes, so we should mostly be testing that...

But anyways, fine for now and thanks for fixing the test!